src/services/message.service.ts

```diff
@@ -75,6 +115,36 @@ export class MessageService {
     return msg;
   }
 
+  private _failedToSend(message: IMessage, reason: string): boolean {
+    message.reply('Sorry the bot cannot send this message because:\n' + reason);
+    return false;
+  }
+
+  private _splitMessage(message: string, argBreakChar: string, numOfMiscChars: number): string[] {
+    const numOfFreeChars = Constants.MaxMessageLength - numOfMiscChars;
+    const breakChar = argBreakChar === '*' ? '(.|\n)' : argBreakChar;
+
+    const messageSplitRegExp: RegExp = new RegExp(
+      `(.|\n){1,${numOfFreeChars}}(${breakChar}|$)`,
+      'gm'
+    );
+
+    return message.match(messageSplitRegExp) || [];
+  }
+
+  private _testMessages(messages: string[], testLength: number, replyLength: number): boolean {
+    let messageCheckCount: number = 0;
+
+    messages.map((message) => {
+      messageCheckCount += message.length;
+    });
+
+    if (testLength != messageCheckCount) {
+      return false;
+    }
+    return true;
+  }
+
```

Comment:
Since this PR is for a method on a `.service` all uses of these methods will be hard coded.
There will no variability in production so, there should never be an error creating the message, thus no reason to have this function

Reply:
It can fail, if the returned text lacks the break character in the 2000 (- template and reply length) character limit.  Then it will parts of the message out.  Granted this case very unlikely, but it is still a possibility.